### PR TITLE
Catalyst refactor

### PR DESCRIPTION
**Pull request by afloresOSI**

### CODE REVIEW

File: content/analytics/data-views/data-view-configure-identifying-fields.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Fields are only eligible if they include one of the following source types listed in the table below. All source types require a field label. Some source types also require having a key defined. The following table lists eligible data sources along with additional requirements for field labels and keys.
+
+| Eligible source type | Field label required? | Key required? |
+|----------------------|-----------------------|---------------|
+| Id                   | ✔                    | &#10006;      |
+| Name                 | ✔                    | &#10006;      |
+| Metadata             | ✔                    | ✔             |
+| Tags                 | ✔                    | ✔             |
+
```
Comment (afloresOSI) [inline content/analytics/data-views/data-view-configure-identifying-fields.md:19]: A simple example of identifying fields would be really useful.

File: content/analytics/data-views/data-view-configure-field-order.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## To edit field order
+
+You can either edit the field order by using page controls or drag and drop.
+
+### [Page controls](#tab/tabid-1)
+
+1. Within a field set, select the field that you want to reorder.
+
+1. Select **Arrow up** ![arrow up](../../_icons/default/arrow-up.svg) or **Arrow down** ![arrow down](../../_icons/default/arrow-down.svg) to move the field to the desired position within the field set.
```
Comment (afloresOSI) [inline content/analytics/data-views/data-view-configure-field-order.md:17]:
```suggestion
1. Select **Arrow up** ![arrow up](../../_icons/default/arrow-up.svg) or **Arrow down** ![arrow down](../../_icons/default/arrow-down.svg) to move the field to the new position within the field set.
```
I really dislike the words "Desire" and "Wish" in documentation.

File: content/analytics/data-views/data-view-configure-field-order.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### [Drag and drop](#tab/tabid-2)
+
+To reorder data fields within a data field set, you can also drag and drop a field to its desired position.
```
Comment (afloresOSI) [inline content/analytics/data-views/data-view-configure-field-order.md:21]:
```suggestion
To reorder data fields within a data field set, you can also drag and drop a field to its new position.
```

File: content/analytics/data-views/data-view-link-fields.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+uid: data-view-link-fields
+---
+
+# Link data fields
+
+Data items included in a data view may have slight differences in property naming, despite those properties representing the same logical thing. For example, data from one equipment manufacturer reports `Temperature`, while another reports `Temp` instead.
+
+Data views can overcome property naming differences by linking these similar properties into a single data field. This applies to stream properties referenced by id or by name, and to stream metadata keys. Asset properties can also be linked.
```
Comment (afloresOSI) [inline content/analytics/data-views/data-view-link-fields.md:9]: The standard for id is "Id".

File: content/analytics/data-views/data-views-manage-data-views.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Edit an existing data view to edit its name and description.
+
+1. In the left pane, select **Analytics** > **Data Views**.
+
+1. Select the data view that you want to edit.
+
+1. Select ![Edit Data View](../../_icons/default/pencil.svg) **Edit Data View**.
+
+	**Getting `Unknown Community`?** This message indicates that the community has at least one query that you have insufficient permissions to edit and save. You must be a member of the community to access its data. Request that a community administrator add you to the community. For more information, see <xref:community-manage-users>.
```
[the remark this answers is not in the document]
Comment (osisoft-mbishop) [reply content/analytics/data-views/data-views-manage-data-views.md:23]: ty!

File: content/analytics/data-views/data-view-configure-field-order.md (hunk 2 above)
Comment (osisoft-mbishop) [reply content/analytics/data-views/data-view-configure-field-order.md:17]: Agreed, but sometimes I can't find a better construction. Thanks for this suggestion--I much prefer it.

File: content/analytics/data-views/data-view-configure-identifying-fields.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/analytics/data-views/data-view-configure-identifying-fields.md:19]: Yeah, agreed. I'll ask Alicia if she can provide one.

**Pull request closed without merging**